packages/replay-next/components/inspector/values/ErrorRenderer.tsx

```diff
@@ -1,14 +1,53 @@
 import { Object as ProtocolObject } from "@replayio/protocol";
+import { useContext } from "react";
 
+import { getObjectPropertySuspense } from "replay-next/src/suspense/ObjectPreviews";
+import { ReplayClientContext } from "shared/client/ReplayClientContext";
+
+import ValueRenderer from "../ValueRenderer";
 import { ObjectPreviewRendererProps } from "./types";
 import styles from "./shared.module.css";
 
 // Renders a protocol ObjectPreview representing an Error with a format of:
 //   ErrorName: Error message...
 //
 // https://static.replay.io/protocol/tot/Pause/#type-ObjectPreview
-export default function ErrorRenderer({ object }: ObjectPreviewRendererProps) {
-  return <span className={styles.Error}>{errorProtocolObjectToString(object)}</span>;
+export default function ErrorRenderer({ object, pauseId }: ObjectPreviewRendererProps) {
+  const replayClient = useContext(ReplayClientContext);
+  const { className } = object;
+
+  const messageProperty = object?.preview?.properties?.find(
+    property => property.name === "message"
+  );
+
+  let errorContent: React.ReactNode = className;
+
+  if (messageProperty) {
+    if (messageProperty.hasOwnProperty("get")) {
+      const getterValue = getObjectPropertySuspense(
+        replayClient,
+        pauseId,
+        object.objectId,
+        "message"
+      );
+      errorContent = (
+        <ValueRenderer
+          context="nested"
+          layout="vertical"
+          pauseId={pauseId}
+          protocolValue={getterValue}
+        />
+      );
```

Comment:
Should we use the `GetterRenderer` for this case instead?

Reply:
I tried that, but it includes the field name, so you end up with something like:

`SyntaxError:  message:  something went wrong`

The `message` field name seems unnecessary - we just want to show the message value itself, and we don't want differing behavior between when there's `messageProperty.value` and a getter.

Reply:
Ah, understood.